test/integration/BPool.t.sol

```diff
@@ -38,6 +41,7 @@ abstract contract PoolSwapIntegrationTest is Test, GasSnapshot {
   // pool amounts
   uint256 public constant DAI_LP_AMOUNT = FOUR_THOUSAND_UNITS;
   uint256 public constant WETH_LP_AMOUNT = ONE_UNIT;
+  uint256 public constant WBTC_LP_AMOUNT = ONE_TENTH_UNIT;
```

Comment:
this is an exorbitant amount of wbtc, shouldn't it break something?

Reply:
well it doesn't, for swaps WBTC is not relevant, and for LPing is just a big amount, it doesn't even check the decimals so it should be fine, we could reduce it to less tho